src/bitmessagecli.py

```diff
@@ -617,59 +645,61 @@ def saveFile(fileName, fileData): #Allows attachments and messages/broadcats to
 
     if not os.path.exists(directory):
         os.makedirs(directory)
-        
-    filePath = directory +'/'+ fileName
+
+    filePath = directory + '/' + fileName
 
     '''try: #Checks if file already exists
         with open(filePath):
             print 'File Already Exists'
             return
     except IOError: pass'''
 
-
-    f = open(filePath, 'wb+') #Begin saving to file
+    f = open(filePath, 'wb+')  # Begin saving to file
```

Comment:
```python
with open(filePath, 'wb+') as f:
    f.write(fileData.decode("base64"))
```

Reply:
Have added it to a ticket for further manual treatment of this file.